dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops

```diff
@@ -228,20 +228,41 @@ UPDATE_DISABLED_FEATURES_ON_CONFLICT=CWWKF0046W: The configuration includes an i
 UPDATE_INDIRECT_CONFLICT_INCOMPATIBLE_FEATURES_SAME_PLATFORM_ERROR.explanation=The server configuration contains an incompatible combination of features for the indicated programming models. This conflict can happen if the two incompatible features are configured directly in the server configuration, or if the configured features include different versions of a feature that are incompatible.
 UPDATE_INDIRECT_CONFLICT_INCOMPATIBLE_FEATURES_SAME_PLATFORM_ERROR.useraction=Try to specify a compatible version of the configured features that caused the conflict in the server.xml file. If no other versions are available, then the configured features are not compatible with each other and cannot be configured at the same time.
 
-UPDATE_MISSING_VERSIONLESS_ENV_VAR=CWWKF0048E: A value could not be found for environment variable PREFERRED_FEATURE_VERSIONS
-UPDATE_MISSING_VERSIONLESS_ENV_VAR.explanation=You have versionless features in your configuration and the PREFERRED_FEATURE_VERSIONS environment variable could not be found
-UPDATE_MISSING_VERSIONLESS_ENV_VAR.useraction=Check that the environment variable, PREFERRED_FEATURE_VERSIONS, exists and is spelled correctly.
+# CWWKF0048E  {0} = the unknown platform value in the environment variable
+UNKNOWN_PLATFORM_VALUE_ENV_VAR=CWWKF0048E: Platform environment variable PREFERRED_PLATFORM_VERSIONS platform value {0} does not contain a valid version.
+UNKNOWN_PLATFORM_VALUE_ENV_VAR.explanation=Values in the platform environment variable must be a known platform. The known platforms are a version of javaee, jakartaee, or microProfile.
+UNKNOWN_PLATFORM_VALUE_ENV_VAR.useraction=Check that the environment variable contains only known platforms.
 
 # CWWKF0049E  {0} = the feature that needs to be specified within PREFERRED_FEATURE_VERSIONS
 UPDATE_MISSING_VERSIONLESS_FEATURE_VAL=CWWKF0049E: The environment variable, PREFERRED_FEATURE_VERSIONS, does not contain a value for {0}
 UPDATE_MISSING_VERSIONLESS_FEATURE_VAL.explanation=For each versionless feature in your configuration, there must be a preferred version value for it within PREFERRED_FEATURE_VERSIONS
 UPDATE_MISSING_VERSIONLESS_FEATURE_VAL.useraction=Add a value in PREFERRED_FEATURE_VERSIONS for the feature
 
+# CWWKF0050E  {0} = the versionless feature that cannot be resolved
+UNRESOLVED_VERSIONLESS_FEATURE=CWWKF0050E: The versionless feature, {0}, could not be resolved
+UNRESOLVED_VERSIONLESS_FEATURE.explanation=Versionless features need a platform to be designated before they can be resolved.
```

Comment:
The message is okay. To simplify it, you might change "platform to be designated" to "designated platform"